cmd/backendcmd/spawn_server.go

```diff
@@ -1,15 +1,18 @@
 // Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
-package cmd
+package backendcmd
 
 import (
 	"context"
 	"fmt"
 
+	this "github.com/ava-labs/avalanche-cli/pkg/app"
 	"github.com/ava-labs/avalanche-cli/pkg/binutils"
 	"github.com/spf13/cobra"
 )
 
+var app **this.Avalanche
```

Comment:
why don't we create app on init? I am not sure which is best, the double pointer is clever but still is a double pointer so generates questions.

Reply:
I changed this to eliminate the double pointer, the structure is largely unchanged.